### PR TITLE
Update policy-csp-storage.md

### PR DESCRIPTION
**Pull request by nieaton**
added "Note: Versions prior to 1903 will not support group policy" to applicable sections

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-storage.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ The following list shows the supported values:
 |Enterprise|Yes|Yes|
 |Education|Yes|Yes|
 
+Note: Versions prior to 1903 will not support group policy
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:131]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

File: windows/client-management/mdm/policy-csp-storage.md [hunk 2]
```diff
@@ -299,6 +305,8 @@ ADMX Info:
...
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:308]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

File: windows/client-management/mdm/policy-csp-storage.md [hunk 3]
```diff
@@ -357,6 +365,8 @@ ADMX Info:
...
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:368]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

File: windows/client-management/mdm/policy-csp-storage.md [hunk 4]
```diff
@@ -421,6 +431,8 @@ ADMX Info:
...
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:434]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

File: windows/client-management/mdm/policy-csp-storage.md [hunk 5]
```diff
@@ -241,6 +245,8 @@ ADMX Info:
...
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:248]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

File: windows/client-management/mdm/policy-csp-storage.md [hunk 6]
```diff
@@ -183,6 +185,8 @@ ADMX Info:
...
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-storage.md:188]:
```suggestion
Note: Versions prior to version 1903 do not support group policy.
```

**Review by denisebmsft — APPROVED** [review]
Thank you @nieaton @JohanFreelancer9

**Pull request merged**